src/components/WorkerPoolQueuesTable.vue

```diff
@@ -19,6 +19,13 @@
       </template>
 
       <p-table v-model:selected="selected" :data="filteredWorkerPoolQueues" :columns="columns">
+        <template #priority-heading>
+          <div class="worker-pool-queues-table__priority-heading">
+            Priority
+            <PriorityDescriptionModal />
+          </div>
```

Comment:
I see something like this duplicated in a few places, should there just be a `WorkerPoolPriority` component? Might make even more sense if you end up needing a place to implement a more generic `HelpIconModal` component

Reply:
+1 mostly because there's some css that could get removed from these components into the PriorityHeading component.